src/tests/tests.rs

```diff
@@ -0,0 +1,14 @@
+use super::Histogram;
+
+#[path = "helpers.rs"]
```

Comment:
OK. What sort of "referring back" do you need to do? It would be nice to avoid these git gymnastics; can you explain more about your workflow?